web/ui/static/js/prom_console.js

```diff
@@ -562,7 +561,7 @@ PromConsole.Graph.prototype.buildQueryUrl = function(expr) {
   var p = this.params;
   return PATH_PREFIX + "/api/v1/query_range?query=" +
     encodeURIComponent(expr) +
-    "&step=" + p.duration / this.graphTd.offsetWidth +
+    "&step=" + Math.floor(p.duration / this.graphTd.offsetWidth * 1000) / 1000; +
```

Comment:
```suggestion
    "&step=" + (Math.floor(p.duration / this.graphTd.offsetWidth * 1000) / 1000) +
```

I guess the `;` wont work here

Reply:
huh, how did I not catch that in testing? Ah, the test case from the user bypasses this.